src/main/java/org/terasology/las/bases/BaseProvider.java

```diff
@@ -53,11 +53,15 @@ public void process(GeneratingRegion region) {
         region.setRegionFacet(BaseFacet.class, facet);
     }
 
-    private Region3i CreateBaseRegionFromVector(Vector3i centerBasePosition) {
-        return Region3i.createFromMinMax(new Vector3i(centerBasePosition.x() - LASUtils.BASE_EXTENT, centerBasePosition.y(), centerBasePosition.z() - LASUtils.BASE_EXTENT), new Vector3i(centerBasePosition.x() + LASUtils.BASE_EXTENT, centerBasePosition.y(), centerBasePosition.z() + LASUtils.BASE_EXTENT));
+    private BlockRegion CreateBaseRegionFromVector(Vector3i centerBasePosition) {
```

Comment:
While we are at it, can we rename this to fit our naming conventions, please?
```suggestion
    private BlockRegion createBaseRegionFromVector(Vector3i centerBasePosition) {
```

Reply:
And do we even need a method like this anymore? Isn't the implementation just this:

```java
return new BlockRegion(centerBasePosition).expand(LASUtils.BASE_EXTENT, 0,  LASUtils.BASE_EXTENT);
```